### PR TITLE
Deployment scripts

### PR DESCRIPTION
**Pull request by syahirAmali**

### CODE REVIEW

File: scripts/deployments/DeploymentChecklist.md [hunk 1]
```diff
@@ -19,6 +19,8 @@
     -   [ ] `TierManager` Magistrate - Deployer
     -   [ ] `GuildXP` Owner
 
+    NOTE: Deployer related accounts will need to be configured within the hardhat config file through a .env
```
Comment (Cosmodude) [inline scripts/deployments/DeploymentChecklist.md:22]: only one account required

File: hardhat.config.ts [hunk 2]
```diff
@@ -61,7 +66,14 @@ const config: HardhatUserConfig = {
         // },
         fuji: {
             url: "https://api.avax-test.network/ext/bc/C/rpc",
-            accounts: [env.DEV_WALLET_PRIVATE_KEY],
+            accounts: [
+                env.DEV_WALLET_PRIVATE_KEY,
+                env.NEXUS_MASTER_PRIVATE_KEY,
+                env.PROFILE_NFT_COUNSELOR_PRIVATE_KEY,
+                env.TAVERN_OWNER_PRIVATE_KEY,
```
Comment (Cosmodude) [inline hardhat.config.ts:73]: this should be updated to one address 
GUILD_MASTER_KEY
GUILD_MASTER_PUBLIC_KEY

File: scripts/deployments/config/nonDeployerAccounts.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { ethers } from "hardhat";
+import { NonDeployerConfigAccounts } from "../../../test/helpers/types";
+
+export const nonDeployerConfigAccounts: NonDeployerConfigAccounts = {
+    // Addresses to be replaced with actual values
+    nexusGuardian: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
+    rewarderSteward: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
+    tavernBarkeeper: "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
```
Comment (Cosmodude) [inline scripts/deployments/config/nonDeployerAccounts.ts:8]: what are these accounts?

**Review by Cosmodude — CHANGES_REQUESTED** [review]

File: deployments/avax/localhost/addresses.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+    "registry": "0x5FbDB2315678afecb367f032d93F642f64180aa3",
+    "account": "0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512",
+    "nexus": "0x8464135c8F25Da09e49BC8782676a84730C318bC",
+    "rewarder": "0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0",
+    "profileNFT": "0x663F3ad617193148711d28f5334eE4Ed07016602",
+    "escrows": {
+        "escrowNative": "0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9",
```
[the remark this answers is not in the document]
Comment (syahirAmali) [reply deployments/avax/localhost/addresses.json:8]: yes from localhost

File: hardhat.config.ts [hunk 5]
```diff
@@ -8,6 +8,11 @@ const env = load({
     INFURA_SECRET_KEY: String,
     DEV_WALLET_PRIVATE_KEY: String,
     ETHERSCAN_API_KEY: String,
+    NEXUS_MASTER_PRIVATE_KEY: String,
+    PROFILE_NFT_COUNSELOR_PRIVATE_KEY: String,
```
[the remark this answers is not in the document]
Comment (syahirAmali) [reply hardhat.config.ts]: changed to master as deployer

File: scripts/deployments/DeploymentChecklist.md (hunk 1 above)
Comment (syahirAmali) [reply scripts/deployments/DeploymentChecklist.md:22]: changed to a single deployer

File: scripts/deployments/config/nonDeployerAccounts.ts (hunk 3 above)
Comment (syahirAmali) [reply scripts/deployments/config/nonDeployerAccounts.ts]: placeholder accounts for addresses to be used, changed to deployers address

File: hardhat.config.ts (hunk 2 above)
Comment (syahirAmali) [reply hardhat.config.ts]: updated to guild master

**Review by Cosmodude — DISMISSED** [review]

**Pull request merged**